nb_lightning_talks

```diff
@@ -40,6 +58,243 @@ html"""
+md"""
+🧠 Most EEG researchers use MATLAB tools
+
+🐢💸 But MATLAB can be slow, expensive and not so versitile
+
+"""
+
+# ╔═╡ 2c34cc63-8bb3-4e73-b8c5-d8002bbb317a
+md"""
+
+![](https://raw.githubusercontent.com/s-ccs/workshop_unfold_2025/547bc4cd27b0b88f18349f8223298a9f9de9b18d/img/EEG_tools.jpg)
+"""
+
+# ╔═╡ da755221-0b2c-47ea-94c0-bb90fca4b7a8
+md"""
+### Data and packages
+"""
+
+# ╔═╡ b939c7c7-f55f-408f-83b7-70084406170e
+md"""
+### Fast rendering ⚡
+"""
+
+# ╔═╡ a75eb514-efff-4449-bf73-02dd4e1de70c
+@bind current_time PlutoUI.Slider(1:50, show_value=true, default=4)
+
+# ╔═╡ 47c31314-388a-411e-b808-e2fef4e06d56
+begin
+    ### Example data
+    timestamps = 1:100  # Example time points
+    
+    ### Create a figure
+    
+    ### Create an observable for the data (to bind it to the slider)
+    #dat_obs = Observable(dat_raw[:, 1, 1])  # Initial data slice for the topoplot
+    	
+    ### Plot the initial topoplot
+    plot_topoplot(dat_raw[:, current_time, 1], positions = positions)
+
+    #dat_obs[] = dat_raw[:, current_time, 1]
+end
+
+
+
+# ╔═╡ f845f097-4457-4c09-8d9e-bbce8de5395f
+md"""
+### Easy mapping
+"""
+
+# ╔═╡ fc519351-da2f-45e0-aff5-36efb04c869f
+begin
+	plot_erp(
+	    results,
+	    mapping = (; color = :coefname => "Conditions"),
+	    axis = (; xlabel = "Time [s]"),
+	)
+end
+
+# ╔═╡ 9adf7983-1f80-47ba-8c6a-0c139241b580
+begin
+	plot_erp(results,
+	    mapping = (; col = :coefname, color = :coefname => "Conditions"),
+	    axis = (; xlabel = "Time [s]"),
+	    stderror = true,
+		significance = significancevalues,
+	)
+end
+
+# ╔═╡ 7b66a975-7436-41ed-bb97-2fab7fead3c3
+md"""
+### Combining Plots Easily
+"""
+
+# ╔═╡ e1b85210-df3a-484a-a7b7-e10079c310ee
+md"""
+Simple combination
+- **Create figure**: A figure `f2` is created with size `(900, 600)`.
+- **Append 3 plots**: Three plots are added using `!`(`plot_erp!` and `plot_butterfly!`) with appropiate data and styling.
+- **Display figure**: The figure `f2` is displayed at the end.
+"""
+
+# ╔═╡ 69eb22de-5ee1-4bbb-bb5e-56f6ea9976be
+begin 
+	f2 = Figure(size = (900, 600))
+	with_theme(theme_ggthemr(:fresh)) do
+	    plot_erp!(f2[1, 1], StatsModels.coeftable(uf_deconv), mapping = (; color = :coefname => "Conditions"))
+	    plot_erp!(
+	        f2[1, 2],
+	        Unfold.effects(Dict(:condition => ["car", "face"]), uf_deconv),
+	        mapping = (; color = :condition => "Conditions"),
+	    )
+	    plot_butterfly!(f2[2, 1:2], dat; positions = positions,
+	        topo_attributes = (; label_scatter = (; markersize = 5)),
+	        topo_axis = (; height = Relative(0.5), width = Relative(0.5)))
+	end
+	f2
+end
+
+# ╔═╡ c9330906-bc37-4f17-900a-d56225870c14
+md"""
+More advanced plot
+- **Assign each subplot to a specific variable* for readability**
+- **Set superlables**
+"""
+
+# ╔═╡ 240fcfc2-489d-4ba3-b981-a04f6fa703e1
+begin 
+    f3 = Figure(size = (1200, 800))  # Adjusting figure size for 4 plots
+    
+    # Assigning subplots using ga, gb, gc, and gd
```

Comment:
is that better readable? I'm not entirely sure :shrug:

I would use the same plot as before, just add that loop.

You could even reuse the variable `f2` and modify it later and plot it a second time, as long as f2 is once displayed first without the labels, and once with them. Or put the labels in comments first and then put them

Reply:
I dont see why you added the topoplot here, 3 plots are clearly enough to show the strength.

What I think is a really cool feature is that you can modify plots after the fact, e.g. you can add a "f[0,1]" plot in front of the others